bdai_ros2_wrappers/bdai_ros2_wrappers/tf_listener_wrapper.py

```diff
@@ -4,25 +4,24 @@
 
 import rclpy
 from geometry_msgs.msg import TransformStamped
-from rclpy import Context
+from rclpy.node import Node
 from rclpy.time import Duration, Time
 from tf2_ros import ExtrapolationException, TransformException
 from tf2_ros.buffer import Buffer
 from tf2_ros.transform_listener import TransformListener
 
-from bdai_ros2_wrappers.node import NodeWrapper
+import bdai_ros2_wrappers.process as process
 
 
 class TFListenerWrapper(object):
     def __init__(
-        self,
-        node_name: str,
-        wait_for_transform: Optional[Tuple[str, str]] = None,
-        cache_time_s: Optional[int] = None,
-        context: Optional[Context] = None,
+        self, node: Node, wait_for_transform: Optional[Tuple[str, str]] = None, cache_time_s: Optional[int] = None
```

Comment:
> is the node argument here primarily for the test cases or is there a use case where that would be appropriate? it seems like a slight departure from the original code there

For test cases and for multi-node use cases. It does depart from the original code because it is no longer a node of its own.

> should that be typed/default as Optional[Node] = None to make it more clear?

Yes! Fixed in https://github.com/bdaiinstitute/ros_utilities/pull/25/commits/82f8445e40905303f70ec7cb908994d833fbb0ea.